### ISSUE TITLE
"ISM330DLC" Not able to get VDD and VDDOUT 1.8 voltage

### ISSUE THREAD

**Issue by Omkardixi** [STMicroelectronics/STMems_Standard_C_drivers#149]
### STEVAL-MKI109V3

STEVAL-MKI182V2

### Type of bug

VDD is 0V while VDDOUT is 3.2V. 

### Describe the bug

I imported the _prj_MKI109V3.ioc file in the STM32 IDE after that I tried to integrate the example code "ism330dlc_read_data_polling.c" in the project and i just flashed the code using j-link Segger. After flashing the code I observed that the VDD was zero but VDDOUT was 3,2V. Instead of using 915 PWM value I tried some other values also but still voltage is not comming on the both the pins. On the other hand I tried to dump the .bin file from the unico-gui package then I observed the 1.8V on both the pins.
 
### Additional context

Add any other context about the problem here. *(ie: hardware description, compiler used, etc...)*

**Comment by yoursanil** [comment]
YES!!!!!! facing the same issue... I am using STEVAL-MKI109V3 along with STEVAL-MKI182V2, whenever i am inserting sensor on to the 24 pin connector the Vdd is getting dropout.
